pom.xml

```diff
@@ -37,14 +37,32 @@
               <!-- stream benchmark under BSD-like license with a few exceptions around benchmarking results -->
               <exclude>tools/bin/src/stream/*</exclude> 
 
-              <!-- PyGreSQL an open-source Python module that interfaces to a PostgreSQL database under the Python license -->
+              <!-- exclude open-source python modules -->
+              <!-- MIT license for open-source python modules -->
               <exclude>tools/bin/pythonSrc/PyGreSQL-4.0/**</exclude>
+              <exclude>tools/bin/pythonSrc/PSI-0.3b2_gp/**</exclude>
+              <exclude>tools/bin/pythonSrc/lockfile-0.9.1/**</exclude>
+              <!-- BSD license for open-source python modules -->
+              <exclude>tools/bin/pythonSrc/pychecker-0.8.18/**</exclude>
+              <exclude>tools/bin/pythonSrc/unittest2-0.5.1/**</exclude>
+              <!-- apache compatible license for open-source python modules -->
+              <exclude>tools/bin/pythonSrc/pycrypto-2.0.1/**</exclude>
+              <!-- Apache compatible open-source python modules under other directories -->
+              <exclude>tools/bin/ext/**</exclude>
+
+              <!-- Apache license for open-source thrift  -->
+              <exclude>depends/thirdparty/thrift/**</exclude>
+
+              <!-- BSD license for open-source googletest -->
+              <exclude>depends/thirdparty/googletest/**</exclude>
 
               <!-- BSD license -->
               <exclude>depends/libyarn/CMake/FindBoost.cmake</exclude>
               <exclude>depends/libyarn/CMake/FindGSasl.cmake</exclude>
               <exclude>depends/libyarn/CMake/FindKERBEROS.cmake</exclude>
+			  <exclude>depends/libyarn/CMake/FindGoogleTest.cmake</exclude>
 			  <exclude>depends/libhdfs3/CMake/FindBoost.cmake</exclude>
+			  <exclude>depends/libhdfs3/CMake/FindGoogleTest.cmake</exclude>
```

Comment:
Where did this file come from? If you wrote it, please add AL header to it. If not -- indicate where it came from.